src/main/java/net/jodah/typetools/TypeResolver.java

```diff
@@ -347,6 +356,58 @@ private static void populateLambdaArgs(Class<?> functionalInterface, final Class
     }
   }
 
+  private static final class MethodRefOffsetResolutionFailed extends RuntimeException {}
+
+  /**
+   * Resolves method ref offset.
+   */
+  private static int resolveMethodRefOffset(ConstantPool constantPool, Class<?> lambdaType) {
```

Comment:
Probably not a bug - just an artifact of a previous refactoring, I think from a pull request. Thanks for pointing this out.